tests/test_operators.py

```diff
@@ -91,6 +94,119 @@ def test_filter_by_values(self):
 
         test_operator(operator=FilterByValues(values={"a": 1}), inputs=inputs, targets=targets, tester=self)
 
+    def test_filter_by_list_of_values(self):
+        inputs = [
+            {"a": 1, "b": 2},
+            {"a": 2, "b": 3},
+            {"a": 3, "b": 4},
+        ]
+
+        targets = [
+            {"a": 2, "b": 3},
+            {"a": 3, "b": 4},
+        ]
+
+        test_operator(
+            operator=FilterByListsOfValues(values={"b": ["3", "4"]}), inputs=inputs, targets=targets, tester=self
+        )
+
+        with self.assertRaises(ValueError) as cm:
+            test_operator(
+                operator=FilterByListsOfValues(values={"b": "5"}), inputs=inputs, targets=targets, tester=self
+            )
+        self.assertEqual(str(cm.exception), "The filter for key ('b') in FilterByListsOfValues is not a list but '5'")
+
+    def test_intersect(self):
+        inputs = [
```

Comment:
suggestion: add tests when `process_every_value` is `True`

Reply:
I disable this feature for intersect because it would be confusing with nested lists.